README.md

```diff
@@ -10,12 +10,13 @@ Asynchronous Apple Push Notification Service client.
 
 
 ```python
-from aapns import connect, Notification, Alert, Production, Localized
-from aapns.config import Priority
+from aapns.api import create_client
+from aapns.config import Priority, Production
+from aapns.models import Notification, Alert, Localized
 
 async def send_hello_world():
-    connection = await connect('/path/to/push/cert.pem', Production)
-    apns_id = await connection.send_notification(
+    client = await create_client('/path/to/push/cert.pem', Production)
```

Comment:
I wonder if renaming the first/main function of the library...
It's a breaking change for the sake of breaking change 🙄 
The rest of the API didn't seem to change much, so why this?
Maybe it's worth adding a deprecated compatibility alias?
Or just hope that almost none uses this library? 😓

Reply:
The signature actually changes too, so I think it's somewhat appropriate. Blindly upgrading to this release will likely end in tears anyway, so failing early and failing fast seems like a decent solution.

Reply:
Uhm... I guess I'm missing something. If I were to trust this README.md, then the only external difference is `connect` → `create_client`...
Then again, perhaps the point is moot, and we are the only users? :)

Reply:
added a CHANGELOG.md (there should've been one anyway imho)